src/scenes/Projects/StatusHistory/ProjectStatusChip.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Chip, Tooltip, Typography } from '@mui/material';
+
+export const ProjectStatusChip = ({ status }: { status: string }) => {
```

Comment:
Decouple the layout here. Component's should not assume layout in any way.
It could be named generically too, like
```tsx
<TextChip>{status}</TextChip>
```